### PR TITLE
[camera] change mediaRecorder.setVideoEncodingBitRate for better video quality

### PR DESCRIPTION
**Pull request by divyanshub024**
closes https://github.com/flutter/flutter/issues/26959

## Description

Changing `mediaRecorder.setVideoEncodingBitRate` from `1024 * 1000` to `3000000` for better quality.

### CODE REVIEW

**Comment by divyanshub024** [comment]
I signed it!

**Comment by Sreejithps** [comment]
I am urgently looking for this fix as we have an upcoming deliverable to a customer.  When with this change be rolled out?
Alternately, any pointers to do a local build?

**Comment by divyanshub024** [comment]
@Sreejithps Check my answer here https://stackoverflow.com/a/54347262/6706175. You have to manually make changes in plugin till it's fixed.

File: packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java [hunk 1]
```diff
@@ -464,7 +464,7 @@ private void prepareMediaRecorder(String outputFilePath) throws IOException {
       mediaRecorder.setOutputFormat(MediaRecorder.OutputFormat.MPEG_4);
       mediaRecorder.setAudioEncoder(MediaRecorder.AudioEncoder.AAC);
       mediaRecorder.setVideoEncoder(MediaRecorder.VideoEncoder.H264);
-      mediaRecorder.setVideoEncodingBitRate(1024 * 1000);
+      mediaRecorder.setVideoEncodingBitRate(3000000);
       mediaRecorder.setAudioSamplingRate(16000);
       mediaRecorder.setVideoFrameRate(27);
       mediaRecorder.setVideoSize(videoSize.getWidth(), videoSize.getHeight());
```
Comment (jonasbark) [inline packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java]: I'm wondering if using existing profiles would be a better option:
`mediaRecorder.setProfile(CamcorderProfile.get(CamcorderProfile.QUALITY_480P));` 
What do you think?

Comment (quentinleguennec) [reply packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java]: I implemented a solution using CamcorderProfiles and matching quality between Android and iOS. Comments and suggestions are welcome.
https://github.com/flutter/plugins/pull/1403

Comment (jonasbark) [reply packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java]: I think that's indeed the way to go!

Comment (leszekkrol) [reply packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java:470]: Any update in this case?

**Pull request closed without merging**